### PR TITLE
New package: lcov-1.13

### PR DESCRIPTION
**Pull request by misery**

### CODE REVIEW

**Comment by misery** [comment]
Well... sourceforge for downloads is crap

**Comment by misery** [comment]
ping?

File: srcpkgs/lcov/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+build_style=gnu-makefile
+depends="perl"
+hostmakedepends="perl"
+short_desc="Front-end for GCC's coverage testing tool gcov"
+maintainer="Andre Klitzing <aklitzing@gmail.com>"
+license="GPL-2"
+homepage="http://ltp.sourceforge.net/coverage/lcov.php"
+distfiles="https://github.com/linux-test-project/${pkgname}/archive/v${version}.tar.gz"
+checksum=3650ad22773c56aaf8c5288e068dd35bd03f57659b6455dc6f8e21451c83b5e8
+
```
Comment (the-maldridge) [inline srcpkgs/lcov/template]: This blank line shouldn't be here.

Comment (misery) [inline srcpkgs/lcov/template]: done

**Comment by renatoaguiar** [comment]
:+1:

**Pull request merged**